controllers/keystoneapi_controller.go

```diff
@@ -1048,6 +1048,14 @@ func (r *KeystoneAPIReconciler) reconcileNormal(
 	}
 	// create Deployment - end
 
+	if instance.Status.ReadyCount > 0 {
```

Comment:
should this be  `instance.Status.ReadyCount == instance.Spec.Replicas`? like all replicas are up using the new account, then we can remove the finalizer.

Reply:
OK, sounds good

Reply:
see https://github.com/openstack-k8s-operators/keystone-operator/pull/390